src/frontend/tsx/Games.tsx

```diff
@@ -87,12 +81,6 @@ export function Games () {
                 key={key}
                 className={`game active ${colorClass}`}
                 title={gameID}
-                //  XX: attemtped to remove setDisplayIndex to see the
-                //      latest move first, but that stuff doesn't come into
-                //      play until you touch the GamePanel
-                //
-                //      removed async () => part of this onClick. maybe
-                //      doubling that is bad?
                 onClick={() => { getArchive(archivedGame); setDisplayIndex(null); setShowingArchive(true) }}>
```

Comment:
`getArchive()` will both scry for the game's moves and use `setDisplayGame()` to display it.